packages/react-components/src/components/kpi/kpiBase.tsx

```diff
@@ -129,7 +130,14 @@ export const KpiBase: React.FC<KPIBaseProperties> = ({
                   }}
                 />
                 <div className='timestamp' data-testid='kpi-timestamp'>
-                  {isLoading ? '-' : new Date(point.x).toLocaleString()}
+                  {isLoading ? (
+                    '-'
+                  ) : (
+                    <DateTime
+                      dateTime={point.x}
+                      pattern='dd/MM/yyyy, HH:mm:ss'
```

Comment:
Can we make these constants? I believe this exists already here - packages/react-components/src/utils/time.ts

Reply:
Also, let's default the DateTime componeent to this so we don't need to pass it everywhere.

Reply:
Done.
Now the default pattern is 'M/d/yy HH:mm:ss' and created a constant ('dd/MM/yyyy, hh:mm:ss' ) as you mentioned to use in react-components.